dts/bindings/usb/atmel,sam0-usb.yaml

```diff
@@ -0,0 +1,30 @@
+---
+title: Atmel SAM0 USB device
+id: atmel,sam0-usb
+version: 0.1
+
+description: >
+    Atmel SAM0 USB in device mode
+
+inherits:
+    !include usb-ep.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
```

Comment:
we don't need type, category, description. They are already declared in usb.yaml included by usb-ep.yaml

Reply:
Done